### ISSUE TITLE
Property `Alias` on `Device1` is sometimes not available

### ISSUE THREAD

**Issue by larsblumberg** [getsenic/gatt-python#8]
```
  File "/srv/senic_hub/venv/lib/python3.5/site-packages/gatt/gatt_linux.py", lin
e 354, in alias
    return self._properties.Get('org.bluez.Device1', 'Alias')
  File "/usr/lib/python3.5/site-packages/dbus/proxies.py", line 70, in __call__
    return self._proxy_method(*args, **keywords)
  File "/usr/lib/python3.5/site-packages/dbus/proxies.py", line 145, in __call__

    **keywords)
  File "/usr/lib/python3.5/site-packages/dbus/connection.py", line 651, in call_
blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.UnknownObject: Method
"Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't
 exist
```

We can catch this exception and return `None` as the device's alias.

**Comment by larsblumberg** [comment]
@grunskis Check this out: http://compuspot.net/general/dbus-error-unknownmethod.html

**Comment by larsblumberg** [comment]
Fixed in f38fc5136c666d1691c74d04cc752c94630acbe0
